### PR TITLE
update the README for archiving this repo

### PR DESCRIPTION
**Pull request by JaydenLiang**

### CODE REVIEW

**Review by jamie-pate — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,60 +1,13 @@
...
+This repository has been archived.
 
-This project contains the code and templates for the **Microsoft Azure** deployment. For deployment on other cloud platforms, visit the relevant repository:
-* The **AliCloud** deployment is in the  [alicloud-autoscale](https://github.com/fortinet/alicloud-autoscale/) repository.
-* The **Amazon AWS** deployment is in the [fortigate-autoscale-aws](https://github.com/fortinet/fortigate-autoscale-aws) repository.
-* The **GCP** deployment is in the [fortigate-autoscale-gcp](https://github.com/fortinet/fortigate-autoscale-gcp) repository.
+The ***FortiGate Autoscale for Azure*** project has been moved to the new location: [https://github.com/fortinet/fortigate-autoscale-azure](https://github.com/fortinet/fortigate-autoscale-azure)
 
-This project is organized in separate node modules:
+The last release of this repository is [3.1.2](https://github.com/fortinet/fortigate-autoscale/releases/tag/3.1.2).
```
Comment (klc178) [inline README.md:7]:
```suggestion
The last release of this repository is [3.1.2](https://github.com/fortinet/fortigate-autoscale/releases/tag/3.1.2) and is for FortiGate for Azure.
```

File: README.md [hunk 2]
```diff
@@ -1,60 +1,13 @@
...
-| 1.0 | Supports auto scaling for PAYG instances only.  | PDFs for AWS and for Azure are available in the 1.0.6 branch.<br/>[deploying-auto-scaling-on-aws-1.0.pdf](https://github.com/fortinet/fortigate-autoscale/blob/1.0/docs/deploying-auto-scaling-on-aws-1.0.pdf)<br/>[deploying-auto-scaling-on-azure-1.0.pdf](https://github.com/fortinet/fortigate-autoscale/blob/1.0/docs/deploying-auto-scaling-on-azure-1.0.pdf) |
-
-# Support
-Fortinet-provided scripts in this and other GitHub projects do not fall under the regular Fortinet technical support scope and are not supported by FortiCare Support Services.
-For direct issues, please refer to the [Issues](https://github.com/fortinet/fortigate-autoscale/issues) tab of this GitHub project.
-For other questions related to this project, contact [github@fortinet.com](mailto:github@fortinet.com).
+The releases in the new location start from [3.3.0](https://github.com/fortinet/fortigate-autoscale-azure/releases/tag/3.3.0).
 
 ## License
+
```
Comment (klc178) [inline README.md:12]:
```suggestion
```

File: README.md [hunk 3]
```diff
@@ -1,60 +1,13 @@
 # FortiGate Autoscale for Azure
 
-A collection of **Node.js** modules and cloud-specific templates which support auto scaling functionality for groups of FortiGate-VM instances on various cloud platforms.
+This repository has been archived.
 
-This project contains the code and templates for the **Microsoft Azure** deployment. For deployment on other cloud platforms, visit the relevant repository:
-* The **AliCloud** deployment is in the  [alicloud-autoscale](https://github.com/fortinet/alicloud-autoscale/) repository.
-* The **Amazon AWS** deployment is in the [fortigate-autoscale-aws](https://github.com/fortinet/fortigate-autoscale-aws) repository.
-* The **GCP** deployment is in the [fortigate-autoscale-gcp](https://github.com/fortinet/fortigate-autoscale-gcp) repository.
+The ***FortiGate Autoscale for Azure*** project has been moved to the new location: [https://github.com/fortinet/fortigate-autoscale-azure](https://github.com/fortinet/fortigate-autoscale-azure)
```
Comment (klc178) [inline README.md:5]:
```suggestion
The ***FortiGate Autoscale for Azure*** project has been moved to a new location: [https://github.com/fortinet/fortigate-autoscale-azure](https://github.com/fortinet/fortigate-autoscale-azure)
```

File: README.md (hunk 1 above)
Comment (klc178) [reply README.md:7]: What is the purpose of including this if this project is being archived?

File: README.md [hunk 4]
```diff
@@ -1,60 +1,13 @@
...
-| 2 | Wait for the stage 1 deployment to complete. |
-| 3 | Deploy the stage 2 template using the same [deployment parameters file](https://github.com/fortinet/fortigate-autoscale/blob/main/azure_template_deployment/templates/deploy_fortigate_autoscale.hybrid_licensing.params.json) with the same parameter values used in step 1.<br><a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2Ffortinet%2Ffortigate-autoscale%2Fmaster%2Fazure_template_deployment%2Ftemplates%2Fdeploy_fortigate_autoscale.hybrid_licensing.json" target="_blank"><img src="http://azuredeploy.net/deploybutton.png"/></a> |
-
-Refer to the [documentation](https://docs.fortinet.com/vm/azure/fortigate/6.4/azure-cookbook/6.4.0/161167/deploying-auto-scaling-on-azure) for descriptions of the deployment parameters file.
-
-## Project development history
```
Comment (klc178) [inline README.md:47]: Related to the previous question. If we are including the link to the last release in this project, we should also include these.
If you agree, I suggest changing the title above to "Project history". Please send back to me for updates.

**Review by klc178 — CHANGES_REQUESTED** [review]

File: README.md [hunk 5]
```diff
@@ -1,60 +1,13 @@
 # FortiGate Autoscale for Azure
```
[the remark this answers is not in the document]
Comment (JaydenLiang) [reply README.md:1]: it's original name is  FortiGate Autoscale for Azure. Should we alter it?

File: README.md (hunk 2 above)
Comment (JaydenLiang) [reply README.md:12]: the last empty line was automatically added per  format suggestion pulgin. I think we can stick with it.

File: README.md (hunk 4 above)
Comment (JaydenLiang) [reply README.md:47]: Users / readers can switch to the 3.1.2 tag branch to see the history whenver they want. Our goal here is to eliminate almost all contents but just two pieces of index for reference, one for the last full version of this repo and the other is the starting point of the new repo.

Comment (JaydenLiang) [reply README.md:47]: btw, everything in the main branch except for the README will be gone at the end.

Comment (klc178) [reply README.md:47]: I understand now. Thanks for the explanation.

File: README.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (klc178) [reply README.md:1]: Either one will work, as this project is being archived. I had made the recommendation as the 1.0 template version of this was for FortiGate Autoscale and covered Azure and AWS. Most recently, the 3.x templates were for Azure only.

**Review by klc178 — APPROVED** [review]

**Pull request merged**